create.js

```diff
@@ -36,9 +37,22 @@ exports.write_3 = function(name, variant, dev) {
 	}
 
 	//Make folder for blank JSON
+	// var newFolderDir = "public/"+name;
+	// if (!fs.existsSync(newFolderDir)){
+	//     fs.mkdirSync(newFolderDir);
+	// }
+
+	// The name you use for the folder creation
 	var newFolderDir = "public/"+name;
+
+	// Ensure the parent directory exists
+	if (!fs.existsSync('public')) {
```

Comment:
Can we make this into a variable that way if this ever needs to change the variable `newFolderDir` and here will be updated automatically.